### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by almosrof120** [community/community#18820] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, This is Al Mosrof Mia. I just now join GitHub Student Developer Pack.

**Comment by melvincwng** [comment]
Hi @khan457, welcome to Github Community!

**Comment by queenofcorgis** [comment, marked as answer]
Welcome @khan457 👋🏼 - we have a thread for students to introduce themselves [here](https://github.com/github-community/education/discussions/12). Come say hello!
